05_swap.md

```diff
@@ -415,27 +415,26 @@ Right now it still makes a difference.
 You remove this `inline` and you
 could get enormous performance difference.
 It could be a factor of 10 for something like swap.
-The problem is that the function call sequences,
+The problem is that the function call sequence,
```

Comment:
This sentence doesn't really make much sense, before or after my edit!

I think he's trying to say that if you don't get any function inlining then repeatedly calling `swap` can be very expensive. But of course these days, the `inline` keyword has essentially no bearing on whether the compiler will inline the function. It's more about avoiding ODR violations. I think that was also true in 2013 when the video was made. Alex's prediction that `inline` wouldn't be necessary 'in 5 years time' had already come true!

Probably one for a footnote? But that's definitely above my pay grade.

Reply:
> This sentence doesn't really make much sense

I think by "sequence" he means "sequence of instructions".

> I think that was also the case in 2013

Just wait until you see `>>` instead of `/ 2`! 

> Probably one for a footnote? 
 
Are you thinking of a footnote about how compilers have changed to no longer need inline? Or do you want to clarify the penalty of function calls for each swap? 

The latter seems pretty clear to me as is. I think the former would be helpful, but I don't know enough to write it. Sounds like it might be at your pay grade?

Reply:
I am going to merge this PR, and we can address the footnote in another.